common/tracing/src/logging.rs

```diff
@@ -94,16 +89,17 @@ pub fn init_global_tracing(app_name: &str, dir: &str, level: &str) -> Vec<Worker
     let jaeger_layer = Some(tracing_opentelemetry::layer().with_tracer(tracer));
 
     // Use env RUST_LOG to initialize log if present.
-    // Otherwise use the specified level.
+    // Otherwise, use the specified level.
     let directives = env::var(EnvFilter::DEFAULT_ENV).unwrap_or_else(|_x| level.to_string());
     let env_filter = EnvFilter::new(directives);
     let subscriber = Registry::default()
+        .with(fmt::layer().with_ansi(atty::is(atty::Stream::Stdout)))
```

Comment:
Should we output our log to `stderr`?

Reply:
I guess users usually use `2>&1` to do that?